### PR TITLE
feat: disable map when using screen reader

### PR DESCRIPTION
**Pull request by rosvik**
resolves https://github.com/AtB-AS/kundevendt/issues/2474

Disables map page when screen reader is enabled, and shows this screen instead:

collage

### CODE REVIEW

**Comment by rosvik** [comment]
I noticed that in light mode, the status bar becomes invisible towards the background. This is also a problem for the map screen in general, where a dark status bar would give more contrast towards the map. 



Do we have any option to set the status bar color?  I tried adding

```tsx
<StatusBar barStyle="dark-content"></StatusBar>
```

to the map page, but this seemed to set the color globally.

File: src/screens/Map/components/MapDisabledForScreenReader.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  const {t} = useTranslation();
+  return (
+    <SafeAreaView style={styles.container}>
+      <ThemedMapImage />
+      <ThemeText type="body__primary--bold" style={styles.header}>
+        {t(MapTexts.disabledForScreenReader.title)}
+      </ThemeText>
+      <ThemeText style={styles.description}>
+        {t(MapTexts.disabledForScreenReader.description)}
+      </ThemeText>
```
Comment (gorandalum) [inline src/screens/Map/components/MapDisabledForScreenReader.tsx:18]: I'll suggest that we add an accessible `View` around all this, with an a11y label reading both the title and description.

**Review by gorandalum — DISMISSED** [review]
I haven't tested this, so it might be working ok, but even though I think it is better to handle this more explicitly.

Normally when people with screen reader select a tab the focus is changed to the header title. On this page there is no header, so we should probably change the focus to the message about the map not being enabled for screen reader users. This can be done with the `useFocusOnLoad` hook.

File: src/screens/Map/components/MapDisabledForScreenReader.tsx (hunk 1 above)
Comment (rosvik) [reply src/screens/Map/components/MapDisabledForScreenReader.tsx]: Updated here 1c95aae72a955775f427cf5e021c2fd92c5dd842.

Comment (gorandalum) [reply src/screens/Map/components/MapDisabledForScreenReader.tsx]: Nice. Tested on Android?

Comment (rosvik) [reply src/screens/Map/components/MapDisabledForScreenReader.tsx]: I don't have a device to test accessibility on android. Is there any known differences between iOS and android here that I should be aware of?

Comment (gorandalum) [reply src/screens/Map/components/MapDisabledForScreenReader.tsx]: Yes 😅 Should try to borrow one from Tor.

Comment (rosvik) [reply src/screens/Map/components/MapDisabledForScreenReader.tsx]: Works well on Android ✅

**Review by tormoseng — APPROVED** [review]
LGTM!

**Pull request merged**